project-metrics/metrics_service/metrics/travis_greenness.py

```diff
@@ -0,0 +1,61 @@
+"""Travis-Greenness metric."""
+
+import datetime
+import sqlalchemy
+from typing import Dict, Text
+
+import db_engine
+from metrics import base
+import models
+
+
+class TravisGreennessMetric(base.PercentageMetric):
+  """A metric tracking the percentage of green Travis builds."""
+
+  def _score_value(self, percentage: float) -> models.MetricScore:
+    if percentage < 0.6:
+      return models.MetricScore.POOR
+    elif percentage < 0.75:
+      return models.MetricScore.MODERATE
+    elif percentage < 0.90:
+      return models.MetricScore.GOOD
+    else:
+      return models.MetricScore.EXCELLENT
+
+  def _count_states(self) -> Dict[models.TravisState, int]:
+    """Counts the number of builds for each relevant build state."""
+    logging.debug('Counting successful builds')
```

Comment:
nit: this can be logging.info - I usually leave logging.debugging for extremely verbose stuff (e.g., if you read thousands of files and want to log each of their names)

Reply:
Done.